src/fiatconnect/index.ts

```diff
@@ -60,6 +65,40 @@ export async function getFiatConnectProviders(
   return providers
 }
 
+/**
+ * Logs in with a FiatConnect provider. Will not attempt to log in if an
+ * unexpired session already exists, unless the `forceLogin` flag is set to `true`.
+ * If the user's wallet is currently locked, will prompt for PIN entry.
+ */
+export async function loginWithFiatConnectProvider(
+  wallet: UnlockableWallet,
+  fiatConnectClient: FiatConnectApiClient,
```

Comment:
where would the fiatConnectClient instance be created?

Reply:
Presumably on the "enter amount" screen? We can also have a function that takes in several FC provider configs and creates a client/logs in to each of them, but still need to decide where that function is called from.